cluster/gce/gci/health-monitor.sh

```diff
@@ -71,10 +71,10 @@ function kubelet_monitoring {
   sleep 120
   local -r max_seconds=10
   local output=""
-  while [ 1 ]; do
+  while true; do
```

Comment:
on modifying: 
```
In ./cluster/gce/gci/health-monitor.sh line 74:
  while [ 1 ]; do
          ^-- SC2161: Instead of '[ 1 ]', use 'true'.
Ref: https://github.com/koalaman/shellcheck/wiki/SC2161
```